lib/node_modules/@stdlib/stats/base/dists/planck/pmf/test/fixtures/python/runner.py

```diff
@@ -0,0 +1,83 @@
+# !/usr/bin/env python
```

Comment:
This file does not match our snippet runner: https://github.com/stdlib-js/stdlib/blob/develop/tools/snippets/test/fixtures/python/runner.py.